Assets/Scripts/Utilities/Spin.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+
+namespace Utilities
+{
+    /// <summary>
+    /// Helper class to rotate an object around the vertical axis.
+    /// </summary>
+    public class Spin : MonoBehaviour
+    {
+        [SerializeField] private float speed;
+
+        private void Update()
+        {
+            Vector3 current = transform.eulerAngles;
+            transform.eulerAngles = new Vector3(current.x, current.y + speed * Time.deltaTime, current.z);
```

Comment:
`Transform.Rotate(0,speed * Time.deltaTime,0)`

Reply:
Rotating like that may produce odd artefacts if localRotation.x and z are not 0. Might need checking first